src/com/thunder_cut/server/ClientInfo.java

```diff
@@ -1,24 +1,31 @@
 /*
- * ClientInformation.java
+ * ClientInfo.java
```

Comment:
You renamed this file.
Use whole words and **avoid** acronyms and abbreviations.

[Naming Conventions](https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367)